client/cz-cardview.html

```diff
@@ -153,6 +153,11 @@
 
       attached: function() {
         registerSource('cz-config', 'cardview-heading', cvHeading => this.set('cardview-heading', (cvHeading ? cvHeading : 'Card Feed')));
+        registerSource('cz-config', 'cardview-columns', cvColumns => {
+          if (cvColumns) {
+            this.set('columns', cvColumns);
```

Comment:
this.columns = cvColumns should do the same thing, if columns is in the properties list. If it isn't, then add it there rather than doing this here.